### PR TITLE
Clean up lodash imports

### PR DESCRIPTION
**Pull request by ddzz**
## Description
Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/219

### CODE REVIEW

File: src/components/CheckboxGroup/CheckboxGroup.jsx [hunk 1]
```diff
@@ -12,12 +12,12 @@ import dispatchAnalyticsEvent from '../../helpers/analytics';
...
 class CheckboxGroup extends React.Component {
   UNSAFE_componentWillMount() {
-    this.inputId = this.props.id || _.uniqueId('errorable-checkbox-buttons-');
+    this.inputId = this.props.id || uniqueId('errorable-checkbox-buttons-');
   }
 
   getMatchingSubSection = (checked, optionValues) => {
     if (checked && this.props.children) {
-      const children = _.isArray(this.props.children)
+      const children = isArray(this.props.children)
```
Comment (bkjohnson) [inline src/components/CheckboxGroup/CheckboxGroup.jsx:20]: We can skip the extra import and use a native function for this:

```suggestion
      const children = Array.isArray(this.props.children)
```

https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_isarray

File: src/components/CheckboxGroup/CheckboxGroup.jsx [hunk 2]
```diff
@@ -68,13 +68,13 @@ class CheckboxGroup extends React.Component {
       requiredSpan = <span className="form-required-span">(*Required)</span>;
     }
 
-    const options = _.isArray(this.props.options) ? this.props.options : [];
+    const options = isArray(this.props.options) ? this.props.options : [];
```
Comment (bkjohnson) [inline src/components/CheckboxGroup/CheckboxGroup.jsx:71]:
```suggestion
    const options = Array.isArray(this.props.options) ? this.props.options : [];
```

File: src/components/Pagination/Pagination.jsx [hunk 3]
```diff
@@ -2,7 +2,7 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import classNames from 'classnames';
-import range from 'lodash/range';
+import { range } from 'lodash';
```
Comment (bkjohnson) [inline src/components/Pagination/Pagination.jsx:5]: Does this make a difference? I don't have a personal preference - happy to go with whichever has the best performance.

File: src/components/RadioButtons/RadioButtons.jsx [hunk 4]
```diff
@@ -20,12 +20,12 @@ class RadioButtons extends React.Component {
...
 
   UNSAFE_componentWillMount() {
-    this.inputId = this.props.id || _.uniqueId('errorable-radio-buttons-');
+    this.inputId = this.props.id || uniqueId('errorable-radio-buttons-');
   }
 
   getMatchingSubSection(checked, optionValue) {
     if (checked && this.props.children) {
-      const children = _.isArray(this.props.children)
+      const children = isArray(this.props.children)
```
Comment (bkjohnson) [inline src/components/RadioButtons/RadioButtons.jsx:28]:
```suggestion
      const children = Array.isArray(this.props.children)
```

File: src/components/RadioButtons/RadioButtons.jsx [hunk 5]
```diff
@@ -78,13 +78,13 @@ class RadioButtons extends React.Component {
       requiredSpan = <span className="form-required-span">(*Required)</span>;
     }
 
-    const options = _.isArray(this.props.options) ? this.props.options : [];
+    const options = isArray(this.props.options) ? this.props.options : [];
```
Comment (bkjohnson) [inline src/components/RadioButtons/RadioButtons.jsx:81]:
```suggestion
    const options = Array.isArray(this.props.options) ? this.props.options : [];
```

File: src/components/Select/Select.jsx [hunk 6]
```diff
@@ -71,11 +71,11 @@ class Select extends React.Component {
     // Calculate options for select
     let reactKey = 0;
     // TODO(awong): Remove this hack to handle options prop and use invariants instead.
-    const options = _.isArray(this.props.options) ? this.props.options : [];
+    const options = isArray(this.props.options) ? this.props.options : [];
```
Comment (bkjohnson) [inline src/components/Select/Select.jsx:74]:
```suggestion
    const options = Array.isArray(this.props.options) ? this.props.options : [];
```

File: src/helpers/fields.js [hunk 7]
```diff
@@ -16,15 +16,11 @@ export function makeField(value, optionalDirty) {
  * Walks through an object hierarchy of fields and marks everything dirty.
  */
 export function dirtyAllFields(field) {
-  if (
-    _.keys(field).length === 2 &&
-    _.has(field, 'value') &&
-    _.has(field, 'dirty')
-  ) {
+  if (keys(field).length === 2 && has(field, 'value') && has(field, 'dirty')) {
```
Comment (bkjohnson) [inline src/helpers/fields.js:19]: We can use the [native `Object.keys(field)`](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_keys). A native `has` isn't as simple.

```suggestion
  if (Object.keys(field).length === 2 && has(field, 'value') && has(field, 'dirty')) {
```

File: src/helpers/fields.js [hunk 8]
```diff
@@ -16,15 +16,11 @@ export function makeField(value, optionalDirty) {
...
     return makeField(field.value, true);
-  } else if (_.isPlainObject(field)) {
-    return _.mapValues(field, value => dirtyAllFields(value));
-  } else if (_.isArray(field)) {
+  } else if (isPlainObject(field)) {
+    return mapValues(field, value => dirtyAllFields(value));
+  } else if (isArray(field)) {
```
Comment (bkjohnson) [inline src/helpers/fields.js:23]:
```suggestion
  } else if (Array.isArray(field)) {
```

**Review by bkjohnson — COMMENTED** [review]
One question and some suggestions on using native functions in order to avoid extra imports.

File: src/components/CheckboxGroup/CheckboxGroup.jsx (hunk 1 above)
Comment (ddzz) [reply src/components/CheckboxGroup/CheckboxGroup.jsx:20]: Good idea, I'll update this in the other files.

File: src/components/Pagination/Pagination.jsx (hunk 3 above)
Comment (ddzz) [reply src/components/Pagination/Pagination.jsx:5]: Shouldn't make a difference as far as I can tell. Only changed this to make all the imports consistent.

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**